app/src/main/resources/eip-client.properties

```diff
@@ -22,3 +22,9 @@ camel.springboot.main-run-controller=true
 # Camel context name
 camel.springboot.name=${spring.application.name}
 # ----------------------------------------------------------------------------------------------------------------------
+
+## --- EIP Client Configuration ----------------------------------------------------------------------------------------
+# This is a comma-separated list of packages to scan for EIP classes (components, routes, etc.)
+# Default is org.openmrs.eip.* which will scan all packages under org.openmrs.eip. However, you can specify a more
+eip.app.scan.packages=org.openmrs.eip.*
```

Comment:
Adds default as to not break existing projects(e.g `eip-odoo-openmrs`). I believe this should configured from the respective route project.

Reply:
Is `org.openmrs.eip.*` optional because not all users need the core EIP entities? And are there mandatory entity classes in `com.ozonehis.eip.*`?

Reply:
> Is org.openmrs.eip.* optional because not all users need the core EIP entities?

Yes, it's optional. For example demo service and EIP Keycloak Superset.

> And are there mandatory entity classes in com.ozonehis.eip.*

Currently, all routes projects within Ozone uses `com.ozonehis.eip.*` as the base package. Plus other common utils bean e.g [ShutdownHandler](https://github.com/ozone-his/eip-client/blob/main/commons/src/main/java/com/ozonehis/eip/utils/ShutdownHandler.java#L19)